### PR TITLE
BCF-1474: db migration for toml config

### PR DESCRIPTION
**Pull request by krehermann**
This is continuing cleanup for the TOML migration.

Delete tables `*_nodes` that are no longer used. 

The downward migration was generated from the existing DDL. There is followup work to cleanup `_chains` tables

### CODE REVIEW

File: core/store/migrate/migrations/0168_drop_persitent_config_storage.sql [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+-- Drop nodes tables
+DROP TABLE IF EXISTS evm_nodes;
+DROP TABLE  IF EXISTS solana_nodes;
+DROP TABLE  IF EXISTS starknet_nodes;
+
+-- Drop cfg column from chains
+ALTER TABLE  IF EXISTS evm_chains DROP COLUMN cfg;
+ALTER TABLE  IF EXISTS solana_chains DROP COLUMN cfg;
+ALTER TABLE  IF EXISTS starknet_chains DROP COLUMN cfg;
```
Comment (jmank88) [inline core/store/migrate/migrations/0168_drop_persitent_config_storage.sql:11]: Do these serve any purpose besides satisfying foreign key constraints? I thought we would be able to drop them.

Comment (krehermann) [reply core/store/migrate/migrations/0168_drop_persitent_config_storage.sql:11]: oh, i'm not sure, i read the tix as only dropping a field from them. let me double check

Comment (jmank88) [reply core/store/migrate/migrations/0168_drop_persitent_config_storage.sql:11]: Oh yeah I wrote it subtly as 'at least remove...' :facepalm:

Comment (krehermann) [reply core/store/migrate/migrations/0168_drop_persitent_config_storage.sql]: updated to drop the chains tables and all the FKs

Comment (krehermann) [reply core/store/migrate/migrations/0168_drop_persitent_config_storage.sql]: reduced scope to only deal with node tables as discussed

**Pull request merged**